examples/backends/vllm/launch/disagg_router_xpu_gdr.sh

```
@@ -0,0 +1,66 @@
+
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)